server/test/durablejobs/jobs/MigratePrimaryApplicantInfoJobTest.java

```diff
@@ -0,0 +1,124 @@
+package durablejobs.jobs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import com.typesafe.config.Config;
+import durablejobs.DurableJobName;
+import io.ebean.DB;
+import io.ebean.Database;
+import io.ebean.Transaction;
+import io.ebean.annotation.TxIsolation;
+import java.time.Instant;
+import models.AccountModel;
+import models.ApplicantModel;
+import models.PersistedDurableJobModel;
+import org.junit.Test;
+import repository.AccountRepository;
+import repository.ResetPostgres;
+import services.WellKnownPaths;
+import services.applicant.ApplicantData;
+import services.settings.SettingsService;
+
+public class MigratePrimaryApplicantInfoJobTest extends ResetPostgres {
+
+  private ApplicantModel createApplicantWithWellKnownPathData(boolean withPaiData) {
+    Database database = DB.getDefault();
+    Transaction transaction = database.beginTransaction();
+    AccountModel account = new AccountModel();
+    account.setEmailAddress("account@email.com");
+    account.save();
+    ApplicantModel applicant = new ApplicantModel();
+    applicant.setAccount(account);
+    ApplicantData applicantData = applicant.getApplicantData();
+    applicantData.putString(WellKnownPaths.APPLICANT_FIRST_NAME, "Jean");
+    applicantData.putString(WellKnownPaths.APPLICANT_MIDDLE_NAME, "Luc");
+    applicantData.putString(WellKnownPaths.APPLICANT_LAST_NAME, "Picard");
+    applicantData.putDate(WellKnownPaths.APPLICANT_DOB, "2305-07-13");
+    applicantData.putString(WellKnownPaths.APPLICANT_PHONE_NUMBER, "5038234000");
+    if (withPaiData) {
+      applicant.setFirstName("Kathryn");
+      applicant.setMiddleName("");
+      applicant.setLastName("Janeway");
+      applicant.setDateOfBirth("2328-05-20");
+      applicant.setPhoneNumber("2066842489");
+      applicant.setEmailAddress("applicant@email.com");
+    }
+    applicant.save();
+    transaction.commit();
+    transaction.close();
+    return applicant;
+  }
+
+  private void runJob() {
+    runJob(false);
+  }
+
+  private void runJob(Boolean setFlag) {
```

Comment:
suggestion (non-blocking): `paiFlagEnabled` or `paiFlagValue` might be a slightly more readable parameter name than `setFlag`?